Cargo.toml

```diff
@@ -32,6 +32,7 @@ serde = "1"
 sha2 = "0.9"
 thiserror = "1"
 tracing = "0.1.37"
+zeroize = "1"
```

Comment:
We specify this dependency with a minor version as well.

Reply:
Done. Is this to avoid automatic upgrades to a breaking version?

Reply:
Yes, since minor version bumps on behalf of the library could break our compilation in the future.